pkg/ebpf/netlink/conntracker_test.go

```diff
@@ -91,13 +91,23 @@ func TestGetUpdatesGen(t *testing.T) {
 	}
 }
 
+func TestTooManyEntries(t *testing.T) {
```

Comment:
What's the outcome of this test?

Reply:
It was a sanity check to make sure we it doesn't crash. I think it's useful to test this code path since we don't usually hit it in our normal operations.